diam/dict/default.go

```diff
@@ -3395,6 +3403,14 @@ var tgpprorfXML = `<?xml version="1.0" encoding="UTF-8"?>
 			<data type="Unsigned32"/>
 		</avp>
 
+		<avp name="Extended-Max-Requested-BW-DL" code="554" must="V"	may="P" must-not="M" may-encrypt="Y" vendor-id="10415">
+			<data type="Unsigned32"/>
+		</avp>
+
+		<avp name="Extended-Max-Requested-BW-UL" code="555" must="V"	may="P" must-not="M" may-encrypt="Y" vendor-id="10415">
+			<data type="Unsigned32"/>
+		</avp>
+
```

Comment:
you don't edit generated files directly, changes are done in xml dictionary files & then .go files are regenerated

Reply:
now it it is autogenerated